server/jobs/onboarding/freckle.js

```diff
@@ -9,8 +10,9 @@ import { getOrCreateProject, assignProject } from '../../modules/freckle/project
  * @returns {undefined}
  */
 export default async (placement) => {
-  const { fellow } = placement;
-  getOrCreateProject(placement.client_name).then((project) => {
+  const { fellow, id: partnerId } = placement;
+  getOrCreateProject(placement.client_name).then(async (project) => {
     if (project.id) assignProject(fellow.email, project.id);
+    saveFreckleAutomation(partnerId, project.id)
```

Comment:
Saving the partner details to the DB should happen inside the `if` statement checking for if `project.id` exist.